tests/test_pad.py

```diff
@@ -111,23 +111,25 @@ def get_info(im: MetaTensor):
         def check(info1, info2, should_be_same):
             aff1, meta1, app_ops1 = info1
             aff2, meta2, app_ops2 = info2
-            # meta and app_ops always same
-            self.assertEqual(meta1, meta2)
```

Comment:
Hi @rijobro I put some tests here (including the inverse). `tests/test_pad.py` and `tests/tests.test_crop_base.py` should work now.  (this `self.assertEqual(meta1, meta2)` is not guaranteed because there is info about applied operations.)